selfdrive/car/subaru/carstate.py

```diff
@@ -13,6 +14,8 @@ def __init__(self, CP):
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
     self.shifter_values = can_define.dv["Transmission"]["Gear"]
 
+    self.angle_rate_calulator = CanSignalRateCalculator(2)
```

Comment:
think it makes more sense to input the frequency. you'd need to know what DT_CTRL is while reading this

Reply:
done